src/vs/base/browser/markdownRenderer.ts

```diff
@@ -547,7 +547,13 @@ export function renderMarkdownAsPlaintext(markdown: IMarkdownString, withCodeBlo
 
 	const html = marked.parse(value, { async: false, renderer: withCodeBlocks ? plainTextWithCodeBlocksRenderer.value : plainTextRenderer.value }).replace(/&(#\d+|[a-zA-Z]+);/g, m => unescapeInfo.get(m) ?? m);
 
-	return sanitizeRenderedMarkdown({ isTrusted: false }, html).toString();
+	let sanitized = sanitizeRenderedMarkdown({ isTrusted: false }, html).toString();
+	if (sanitized.includes('&lt;-')) {
```

Comment:
Does this only happen for `<-` or does plain old `<` on its own also get encoded? 

Also you can skip the `includes` call and just use `replaceAll`

Reply:
it happens for all of the unescapeInfo